docs/source/tutorial_material/tutorial.ipynb

```diff
@@ -167,19 +163,27 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Inference\n",
+    "### Using inference\n",
     "\n",
-    "Now we can now use EPI to infer the parameter distribution from the data.By default, the `inference` method uses Markov chain Monte Carlo sampling (this can be changed using the inference_type argument). `inference` returns a tuple containing samples from the parameter Markov chain $y_i$, the corresponding data points $q_i = s(y_i)$, the estimated densities $\\Phi_Q (q_i)$ scaled by a constant $c$, and a `ResultManager` object that can be used to load and manipulate the results of EPI."
+    "Now we can now use `eulerpi` to infer the parameter distribution from the data. By default, the `inference` method uses Markov chain Monte Carlo (MCMC) sampling (this can be changed using the inference_type argument). `inference` returns a tuple containing three dicts with the samples from the parameter Markov chain $q_i$, the corresponding data points $y_i = s(q_i)$, the estimated densities $\\Phi_Q (q_i)$ scaled by a constant $c$, as well as a `ResultManager` object that can be used to load and manipulate the results of EPI."
```

Comment:
is $q_i$ a single sample? I yes, would it be better to write it after the samples?